Library/Homebrew/formula_auditor.rb

```diff
@@ -582,15 +582,15 @@ def audit_eol
       metadata = SharedAudits.eol_data(name, formula.version.major)
       metadata ||= SharedAudits.eol_data(name, formula.version.major_minor)
 
-      return if metadata.blank? || metadata["eol"] == false
+      return if metadata.blank? || metadata["eol"].blank?
```

Comment:
```suggestion
      return if metadata.blank? || (eol_date = metadata["eol"]).blank?
```
Might be easier for reuse below.